packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -175,16 +175,17 @@ describe('Set Default Org', () => {
     orgListStub = sinon.stub(OrgList.prototype, 'updateOrgList');
     quickPickStub = sinon.stub(vscode.window, 'showQuickPick');
   });
-  afterEach(() => {
-    orgListStub.restore();
-    quickPickStub.restore();
-  });
 
   it('should return Cancel if selection is undefined', async () => {
-    orgListStub.returns(orgList);
-    quickPickStub.returns(undefined);
-    const response = await setDefaultOrg();
-    expect(response.type).to.equal('CANCEL');
+    try {
```

Comment:
These tests are a bit fragile and once one fails it causes all of them to fail because the stubs didn't get restored.

Reply:
Ah so the afterEach doesn't run if a test fails?

Reply:
That's correct, the only config we could use would be to set `--bail` on the mocha options but that would make the test suite stop on the first error. Didn't use `--bail` since there is value in executing the whole test suite when doing significant changes.

Reply:
Cool, makes sense.

Reply:
So, I understand the reasoning for this change but I don't quite understand why it was necessary? I thought Mocha was supposed to run the afterEach regardless of whether or not the test passed? I can see the executeCommand stub needing a catch where its being used since it's not used everywhere.

Reply:
That's not how mocha behaves when it's configured the way we've done it for our test suites. We can change the configuration later if we think we need to.